### ISSUE TITLE
💅 Biome is adding a comma in the last line inside ReactDOM.createRoot.render method in main.tsx file when executing npx biome check --apply ./src

### ISSUE THREAD

**Issue by CarmeloRicarte** [biomejs/biome#2906]
### Environment information

```bash
CLI:
  Version:                      1.7.1
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v18.15.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "npm/9.5.0"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Linter:
  Recommended:                  true
  All:                          false
  Rules:
Workspace:
  Open Documents:               0
```

### Rule name

no specific rule added

### Playground link

https://codesandbox.io/p/devbox/biomejs-bug-react-render-xtw6n7

### Expected result

It should not add this comma after </React.StrictMode> like in this example:
```tsx
import React from "react";
import ReactDOM from "react-dom/client";
import App from "./App.tsx";
import "./index.css";

ReactDOM.createRoot(document.getElementById("root")!).render(
  <React.StrictMode>
    <App />
  </React.StrictMode>
);

```

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Interestingly, Prettier has the same issue.

**Comment by ah-yu** [comment]
It seems they behave differently when setting `trailing_comma` to es5. [playground](https://biomejs.dev/playground/?lineWidth=120&indentStyle=space&quoteStyle=single&trailingComma=es5&semicolons=as-needed&attributePosition=multiline&bracketSameLine=true&lintRules=all&code=aQBtAHAAbwByAHQAIABSAGUAYQBjAHQAIABmAHIAbwBtACAAIgByAGUAYQBjAHQAIgA7AAoAaQBtAHAAbwByAHQAIABSAGUAYQBjAHQARABPAE0AIABmAHIAbwBtACAAIgByAGUAYQBjAHQALQBkAG8AbQAvAGMAbABpAGUAbgB0ACIAOwAKAGkAbQBwAG8AcgB0ACAAQQBwAHAAIABmAHIAbwBtACAAIgAuAC8AQQBwAHAALgB0AHMAeAAiADsACgBpAG0AcABvAHIAdAAgACIALgAvAGkAbgBkAGUAeAAuAGMAcwBzACIAOwAKAAoAUgBlAGEAYwB0AEQATwBNAC4AYwByAGUAYQB0AGUAUgBvAG8AdAAoAGQAbwBjAHUAbQBlAG4AdAAuAGcAZQB0AEUAbABlAG0AZQBuAHQAQgB5AEkAZAAoACIAcgBvAG8AdAAiACkAIQApAC4AcgBlAG4AZABlAHIAKAAKACAAIAA8AFIAZQBhAGMAdAAuAFMAdAByAGkAYwB0AE0AbwBkAGUAPgAKACAAIAAgACAAPABBAHAAcAAgAC8APgAKACAAIAA8AC8AUgBlAGEAYwB0AC4AUwB0AHIAaQBjAHQATQBvAGQAZQA%2BAAoAKQA7AA%3D%3D)

**Comment by ematipico** [comment]
I believe this is expected and it's not a bug.

When the list of parameters breaks on multiple lines, the formatter always adds a leading comma for the last parameter. In this case, the last is also the first.

However, the es5 trialing comma is definitely buggy

**Comment by Conaclos** [comment]
The ES5 case seems to be fixed.
